src/backend/storeManagers/gog/library.ts

```diff
@@ -696,19 +696,6 @@ export async function getInstallInfo(
     }
   }
   installInfoStore.set(installInfoStoreKey, info)
-  if (!info) {
```

Comment:
a few lines above we do `const info = { ..... }`, I couldn't find any way this could be false since it's defined right before this line